cluster/up.sh

```diff
@@ -46,6 +46,9 @@ elif [[ $image == $OPENSHIFT_IMAGE ]]; then
     ./cluster/.kubectl config set-cluster node01:8443 --insecure-skip-tls-verify=true
 fi
 
+# Temporary until image is updated with provisioner that sets this field
+$gocli ssh node01 -- sudo sysctl -w user.max_user_namespaces=1024
```

Comment:
can you add a comment describing the purpose of this action?

Reply:
It is decsribed in PR https://github.com/kubevirt/kubevirtci/pull/55.
This configuration is requierd by buildah tool which is used inorder to build and push images to registry

Reply:
can you add that this is a requirement for buildah tool and should be removed once the PR is in?
Otherwise we will definitely forget this line exists and won't get rid of it.

Reply:
Sure